next/components/forms/simple-components/ClipboardCopy.tsx

```diff
@@ -0,0 +1,20 @@
+import ContentCopy from '@assets/images/content_copy.svg'
+import { useCopyToClipboard } from 'usehooks-ts'
+
+const ClipboardCopy = ({ copyText }: { copyText: string }) => {
+  const [_, copy] = useCopyToClipboard()
+  return (
+    <button
+      type="button"
+      onClick={() => {
+        copy(copyText)
+          .then(() => {})
+          .catch(() => {})
+      }}
```

Comment:
we did not understand each other the first time - I meant this

```suggestion
      onClick={() => copy(copyText)}
```